resources-linux/k8s/datasource_logs_utils.bash

```diff
@@ -84,7 +84,7 @@ function build_logzio_logs_region_helm_set {
     send_log_to_logzio "$LOG_LEVEL_DEBUG" "$message" "$LOG_STEP_TRACES" "$LOG_SCRIPT_TRACES" "$func_name" "$AGENT_ID" "$PLATFORM" "$SUB_TYPE" "$CURRENT_DATA_SOURCE"
     write_log "$LOG_LEVEL_DEBUG" "$message"
 
-    local helm_set=" --set logzio-logs-collector.secrets.LogzioRegion=$region"
+    local helm_set=" --set logzio-logs-collector.secrets.logzioRegion=$region"
```

Comment:
is it with lower letter `logzioRegion` on purpose? (I remember a ticket about something similar)

Reply:
It was handled in the chart but not the agent.
Released in `logzio-logs-collector` v1.0.4

Reply:
It was handled in the chart but not the agent.
Released in `logzio-logs-collector` v1.0.4